tools/peview/peexceptprp.c

```diff
@@ -166,7 +166,80 @@ VOID PvEnumerateExceptionEntries(
     }
     else if (imageMachine == IMAGE_FILE_MACHINE_ARM64)
     {
-        /* Todo */
+        for (ULONG i = 0; i < exceptions.NumberOfEntries; i++)
+        {
+            PPH_IMAGE_RUNTIME_FUNCTION_ENTRY_ARM64 entry = PTR_ADD_OFFSET(exceptions.ExceptionDirectory, i * sizeof(PH_IMAGE_RUNTIME_FUNCTION_ENTRY_ARM64));
+            INT lvItemIndex;
+            PPH_STRING symbol;
+            PPH_STRING symbolName = NULL;
+            WCHAR value[PH_INT64_STR_LEN_1];
+            IMAGE_ARM64_RUNTIME_FUNCTION_ENTRY_XDATA* exceptionData;
+
+            PhPrintUInt64(value, ++count);
+            lvItemIndex = PhAddListViewItem(ListViewHandle, MAXINT, value, entry);
+
+            PhPrintPointer(value, UlongToPtr(entry->BeginAddress));
+            PhSetListViewSubItem(ListViewHandle, lvItemIndex, 1, value);
+
+            ULONG functionLength = entry->FunctionLength << 2;
```

Comment:
I think we should move accessing the packed `FunctionLength` into the case where we know the entry is in the packed form. Especially since I'm asking for a null pointer check elsewhere.